UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs

```diff
@@ -74,6 +75,18 @@ public static void CloneSelf()
 			playerBody.ApplyDamageAll(null, 2, AttackType.Internal, DamageType.Clone, false);
 		}
 
+		[ConsoleMethod("clearProtips", "Clears all saved states of protips.")]
+		public static void ClearAllProtips()
+		{
+			ProtipManager.Instance.ClearSaveState();
+		}
+
+		[ConsoleMethod("ShowFirstTimeExpScreen", "Shows the player experience screen.")]
```

Comment:
We have to decide on a naming convention for the commands. 
Currently there seems to be all of the following coexisting:
- ``kebab-case``
- ``PascalCase``
- ``lowercase``
- ``camelCase`` 

This is only confusing and users will have to remember which naming convention each command is using.

Reply:
> We have to decide on a naming convention for the commands.

no, we do not.

Reply:
i read that as "we have a naming convention" bruh

but yeah, we do need it. all lower case seems better imo

Reply:
``alllowercase`` can get messy when there are many words. I think ``kebab-case`` looks good and is also the convention you used in the other pr.

Reply:
done